Source/Core/VideoCommon/BPFunctions.cpp

```diff
@@ -74,13 +75,26 @@ void SetScissor()
 
 void SetViewport()
 {
-  s32 xoff = bpmem.scissorOffset.x * 2;
-  s32 yoff = bpmem.scissorOffset.y * 2;
-  float x = g_renderer->EFBToScaledXf(xfmem.viewport.xOrig - xfmem.viewport.wd - xoff);
-  float y = g_renderer->EFBToScaledYf(xfmem.viewport.yOrig + xfmem.viewport.ht - yoff);
+  const s32 xoff = bpmem.scissorOffset.x * 2;
+  const s32 yoff = bpmem.scissorOffset.y * 2;
+  float raw_x = xfmem.viewport.xOrig - xfmem.viewport.wd - xoff;
+  float raw_y = xfmem.viewport.yOrig + xfmem.viewport.ht - yoff;
+  float raw_width = 2.0f * xfmem.viewport.wd;
+  float raw_height = -2.0f * xfmem.viewport.ht;
+  if (g_ActiveConfig.UseGPUTextureDecoding())
```

Comment:
Shouldn't this be UseVertexRounding() ?

Reply:
Yes.  Not sure how I messed that up; I thought I tested again after merging the settings, but maybe I only tested the vertex rounding issue and not the viewport rounding one?